archive/global.php

```diff
@@ -212,7 +212,7 @@
 // Load Limiting - DIRECTORY_SEPARATOR checks if running windows
 if(DIRECTORY_SEPARATOR != '\\')
 {
-	if($uptime = @exec('uptime'))
+	if(function_exists('uptime') ? $uptime = @exec('uptime') : false)
```

Comment:
I suggest moving `function_exists('uptime')` to the other if above to improve readability.